libntfs-3g/index.c

```diff
@@ -2185,6 +2197,7 @@ int ntfs_index_rm(ntfs_index_context *icx)
 	}
 out:
 	return ret;
+sweep_entries:
```

Comment:
Any reason why you have added sweep_entries goto instead of err_out here ?

Reply:
I'll remove it. actually I wanna to sweep all entries from those corrupted entry to the end entry, but not implemented.